conf.py

```diff
@@ -361,6 +361,6 @@
 
 extlinks = {
     'enterprise-plugins-tar': ( 'https://downloads.graylog.org/releases/graylog-enterprise/plugin-bundle/tgz/graylog-enterprise-plugins-%s.tgz', None),
-    'integrations-plugin-jar': ( 'https://downloads.graylog.org/releases/graylog-integrations/jar/graylog-plugin-enterprise-integrations-%s.jar', None),
-    'integrations-plugin-deb': ( 'https://downloads.graylog.org/releases/graylog-integrations/deb/graylog-plugin-enterprise-integrations-%s.deb', None),
-    'integrations-plugin-rpm': ( 'https://downloads.graylog.org/releases/graylog-integrations/rpm/graylog-plugin-enterprise-integrations-%s-1.noarch.rpm', None), }
+    'integrations-plugin-jar': ( 'https://downloads.graylog.org/releases/graylog-integrations/jar/graylog-plugin-integrations-%s.jar', None),
+    'integrations-plugin-deb': ( 'https://downloads.graylog.org/releases/graylog-integrations/deb/graylog-plugin-integrations-%s.deb', None),
+    'integrations-plugin-rpm': ( 'https://downloads.graylog.org/releases/graylog-integrations/rpm/graylog-plugin-integrations-%s-1.noarch.rpm', None), }
```

Comment:
Please include links both for `enterprise-integrations` and `integrations` here. This will allow the user to download the DEB, RPM, or JAR for each.

Like this:

```suggestion
    'integrations-plugin-rpm': ( 'https://downloads.graylog.org/releases/graylog-integrations/rpm/graylog-plugin-integrations-%s-1.noarch.rpm', None),
    'enterprise-integrations-plugin-jar': ( 'https://downloads.graylog.org/releases/graylog-integrations/jar/graylog-plugin-enterprise-integrations-%s.jar', None),
    'enterprise-integrations-plugin-deb': ( 'https://downloads.graylog.org/releases/graylog-integrations/deb/graylog-plugin-enterprise-integrations-%s.deb', None),
    'enterprise-integrations-plugin-rpm': ( 'https://downloads.graylog.org/releases/graylog-integrations/rpm/graylog-plugin-enterprise-integrations-%s-1.noarch.rpm', None), }
```